domapping/cli.py

```diff
@@ -81,14 +89,23 @@ def schema_to_mapping_cli(schema, output, config, indent):
 @click.argument('output', type=click.File('w'), default='-')
 @click.option('--indent', '-i', default=4, type=click.INT,
               help='Output template indentation step.')
-@click.option('--type', '-t', default='type', type=click.STRING,
+@click.option('--type', '-t', type=click.STRING,
               help='Root type name used in jinja block names.')
 def mapping_to_jinja_cli(mapping, output, indent, type):
     """Generate jinja template from Elasticsearch mapping."""
     parsed_mapping = json.load(mapping)
 
-    template = mapping_to_jinja(parsed_mapping, type, indent=indent)
-    output.write(template)
+    if 'mappings' in parsed_mapping:
+        result = []
+        for item in parsed_mapping:
+            template = mapping_to_jinja(parsed_mapping, item, indent=indent)
```

Comment:
I think it would be better if by default the type name was the one already in the mappings dict.
What do you think of this workflow?:
1/ The input file is a doctype mapping (no root `mappings`) => `--type` option is used and is by default equal to `"type"`
2/ The input file is a `mappings` file
  2.1/ The input file contains only one doctype mapping => the name of the doctype in the input file is used if the option `--type` is not set, otherwise the option overrides the type name.
  2.2/ The input file contains multiple doctypes => the name used is the one provided in the input file. The command fails if the option `--type` is set